pcdsdevices/lamp_motion.py

```diff
@@ -65,7 +65,7 @@ class LAMPMagneticBottle(BaseInterface, GroupDevice):
 
     # Motor components
     gas_needle_x = Cpt(BeckhoffAxis, ':MMS:01', kind='normal')
-    gas_needle_y = Cpt(BeckhoffAxis, ':MMS:02', kind='normal')
+    gas_needle_y = Cpt(BeckhoffAxisEPS, ':MMS:02', kind='normal')
```

Comment:
Why not the other gas needles?  are we sure they won't be needing EPS?

Reply:
yes. Only this axis has EPS feature.

Reply:
This axis has EPS across the PLC(lamp-motion to lamp-vac). Others do not need EPS so far.

Reply:
Sounds good.  I left an approval so we're good to go here then